src/components/Task/Task.js

```diff
@@ -0,0 +1,26 @@
+import { useDrag } from 'react-dnd';
+import IconButton from '../IconButton/IconButton';
+import styles from './Task.module.scss';
+
+export default function Task({ task, removeTask }) {
+  const [{ isDragging }, drag] = useDrag(() => ({
+    type: 'task',
+    item: { id: task.id, done: task.done },
+    collect: (monitor) => ({
+      isDragging: !!monitor.isDragging(),
+    }),
+  }));
+  return (
+    <li
+      className={task.done ? styles.task + ' ' + styles.done : styles.task}
+      ref={drag}
+      style={{
+        opacity: isDragging ? 0.5 : 1,
```

Comment:
I would also recommend using `classNames` to 'style' due to specificity issues with inline styles. Do you think it overcomplicates the class name property? no problem - use `clsx` or 'classnames'.

Reply:
Thanks for info, I will avoid inline styles from now on.